GOOLCodeGen/GOOLCode.cabal

```diff
@@ -1,5 +1,5 @@
 Name:		GOOLCode
-Version:	0.0.1
+Version:	0.0.2
```

Comment:
So `GOOLCodeGen` was outside of the `code/` hierarchy to allow independent experimentation. It doesn't make sense to have 3 copies (i.e. the one under `New`, this one, and the one under `code/`).